CMakePCHCompiler.cmake

```diff
@@ -76,16 +76,7 @@ function(target_precompiled_header) # target [...] header
 				ABSOLUTE
 				)
 			get_filename_component(win_pch "${target_dir}/${header}.pch" ABSOLUTE)
-			get_filename_component(win_header "${header}" ABSOLUTE)
-		else()
-			# HACK: This lets the user include the header in their project
-			# files. Because set_target_properties applies in both targets
-			# GCC will error if the -include directive file is nonexistent.
-			get_filename_component(sym_target "${header}" ABSOLUTE)
-			#WTF??? Must get \ escaped spaces out of both paths for Cygwin.
-			set(args "-E create_symlink \"${sym_target}\"; \"${target_dir}/${header}\"")
-			separate_arguments(args)
-			add_custom_target(${target}.pch.h COMMAND ${CMAKE_COMMAND} ${args})
+			get_filename_component(win_header "${header}" ABSOLUTE)		
```

Comment:
Okay, if this does undo the effort, can we squash the commits?

Reply:
"We" can squash the commits... unfortunately "I" don't know how, and don't want to learn :)